README.md

```diff
@@ -22,7 +22,7 @@ And sorry, not the even newer [Paint 3D][].
 [Paint 3D]: https://www.microsoft.com/en-us/store/p/paint-3d-preview/9nblggh5fv99
 
 Windows 95, 98, and XP were the golden years of paint.
-You had a tool box and a color box, a foreground color and a background color,
+You had a toolbox and a color box, a foreground color and a background color,
```

Comment:
I like the symmetry of having a space here, and having each of the four things I'm listing be two-noun phrases, plus this is how MS Paint calls it (e.g. in the View menu)